src/components/Button/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import classnames from 'classnames';
+
+import style from './Button.module.scss';
+
+type ButtonProps =
```

Comment:
Nice. Remember this pattern. It might be helpful in the future :)). If you need to base this "condition" on something other than string constants, it is a bit more annoying